CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Changelog
 
 
+## [1.5.4](https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/compare/1.5.3...v1.5.4) (2022-06-23)
+
+
+### Dependencies
+
+* Bump google-cloud-spanner-jdbc to 2.7.3 ([4ad5262](https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/commit/4ad52625bdf5425ea98f8c2fa1da4a1fd897fd4d))
+* Bump hibernate.version to 5.6.9.Final ([4ad5262](https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/commit/4ad52625bdf5425ea98f8c2fa1da4a1fd897fd4d))
+* bump maven-surefire-plugin from 3.0.0-M6 to 3.0.0-M7 ([#428](https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/issues/428)) ([03c1787](https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/commit/03c178723a3e63bc694b7f2c5f0fa32f65744d5b))
+* bump spring-boot-version from 2.7.0 to 2.7.1 ([#436](https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/issues/436)) ([d00f9de](https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/commit/d00f9de0c555ac5302521fbe599e65285dbdfa50))
```

Comment:
Do these have to get manually removed?

Reply:
yes, gotta remove them manually this time.